filters-share/extended_css_ELEMHIDE.txt

```diff
@@ -1,4 +1,4 @@
-namu.wiki#?#article div:has(> div.wiki-paragraph) ~ div:has(table > tbody > tr > td[class] > div > img[src*="w.namu.la/"])
+namu.wiki#?#article div:has(a[href*="/w/%EB%B6%84%EB%A5%98:"]):has(> *:contains(분류)):not([class]) ~ div:has(table > tbody > tr > td[colspan]:not([style]) div:has(img[src*="ww.namu.la/s/"][class]):not(:has(div.namuwiki-toc-ad)))
```

Comment:
Is this right?

```suggestion
namu.wiki#%#div:has(> table > tbody > tr > td span > img[src^="//ww.namu.la/s/"])
```

Reply:
`#%#`?

Reply:
ah wait, found it.

Reply:
Is it possible to change the domain of PowerLink ad to make List-KR block incorrectly a picture?

Reply:
Changed just like previous one does.

Reply:
Hmm... I will add your rule too.

Reply:
Sorry, I don't get what actually you want to change. Please, explain me if you mind it. I'll try to work better in future.

Reply:
Never mind. I can remove your suggested rule in the future when it causes an issue.